crates/matrix-sdk-ui/src/timeline/event_item/content.rs

```diff
@@ -389,7 +389,7 @@ pub type BundledReactions = IndexMap<String, ReactionGroup>;
 
 // The long type after a long visibility specified trips up rustfmt currently.
 // This works around. Report: https://github.com/rust-lang/rustfmt/issues/5703
-type ReactionGroupInner = IndexMap<(Option<OwnedTransactionId>, Option<OwnedEventId>), OwnedUserId>;
+type ReactionGroupInner = IndexMap<EventItemIdentifier, ReactionSenderData>;
```

Comment:
We could try inlining the type alias into the one above now, see if rustfmt still does weird things.